.travis.yml

```diff
@@ -31,7 +32,6 @@ env:
 addons:
   apt:
     packages:
-      - oracle-java9-installer
       - oracle-java8-installer
```

Comment:
do we still need java 8 **installer**?

Reply:
YES we need, we compile by Java8 to test on Java9.